isobus/src/can_message.cpp

```diff
@@ -39,11 +39,36 @@ namespace isobus
 		return source;
 	}
 
+	bool CANMessage::has_valid_source_control_function() const
+	{
+		return nullptr != source;
+	}
+
 	std::shared_ptr<ControlFunction> CANMessage::get_destination_control_function() const
 	{
 		return destination;
 	}
 
+	bool CANMessage::has_valid_destination_control_function() const
+	{
+		return nullptr != destination;
+	}
+
+	bool CANMessage::is_destination_global() const
```

Comment:
We might want to rename this to 
```suggestion
	bool CANMessage::is_broadcast() const
```